WikipediaUnitTests/Code/MWKLanguageLinkControllerTests.m

```diff
@@ -47,15 +47,15 @@ - (void)testSaveSelectedLanguageUpdatesTheControllersFilteredPreferredLanguages
     //    NSAssert(![[self preferredLanguageCodes] containsObject:@"test"],
```

Comment:
Is it worth adding a test for these altcodes? Don't need one strictly speaking, but could be useful...

Reply:
Yeah, agreed, it would be nice. I'm having a hard time getting `MWKLanguageLinkController` to think the OS languages have "nb" in there, which would be the best test. I'm running low on time today but I can look into it next week if necessary.